content/shmem_broadcast.tex

```diff
@@ -72,12 +72,15 @@
     needed to ensure this): The \VAR{pSync} array on all \ac{PE}s in the
     \activeset{} is not still in use from a prior call to a broadcast routine.  The
     \dest{} array on all \ac{PE}s in the \activeset{} is ready to accept the
-    broadcast data.
+    broadcast data. \newtext{The data in the \source{} array on the root \ac{PE} is ready to be broadcasted.}
```

Comment:
"Before any PE calls a broadcast routine ... the data in the source array on the root PE is ready to be broadcasted."

I thought that we decided against this semantic?  IIUC, this will effectively require a barrier before every broadcast routine, which is definitely going to break backward compatibility.